src/run.ts

```diff
@@ -80,7 +54,7 @@ var walkSync = function (dir, filelist, fileToFind) {
 };
 
 async function downloadHelm(version: string): Promise<string> {
-    if (!version) { version = await getStableHelmVersion(); }
+    if (!version) { version = await getLatestHelmVersionFor("v3"); }
```

Comment:
Should we feature-flag this change to avoid any possible breakage? (Using some env variable perhaps?)

Reply:
Sounds good. Will do that.